ddtrace/tracer/tracer.go

```diff
@@ -265,6 +265,9 @@ func (t *tracer) StartSpan(operationName string, options ...ddtrace.StartSpanOpt
 		Start:        startTime,
 		taskEnd:      startExecutionTracerTask(operationName),
 		noDebugStack: t.config.noDebugStack,
+		Meta: map[string]string{
```

Comment:
Can we move this a bit higher up? By two lines. Just so we don't mix exported and unexported fields. Let's leave the unexported ones at the end.

Reply:
Resolved to move it to down to check if the hostname is not an empty string